tests/functions/test_auth0.py

```diff
@@ -44,6 +44,43 @@ def mock_function_ret_val(function_name, return_value):
     save_logs.assert_called_once_with(logs)
 
 
+def test_get_last_auth0_log_id(monkeypatch):
+    """Test getting last log id"""
+
+    _get_log_bucket = MagicMock()
+    bucket = MagicMock()
+    get_blob = MagicMock()
+    bucket.get_blob.return_value = get_blob
+    list_blobs = MagicMock()
+    bucket.list_blobs.return_value = list_blobs
```

Comment:
I think either `bucket.list_blobs.return_value = None` or `bucket.list_blobs.return_value = []` is a more accurate mock return value for this function.